activerecord/test/cases/date_time_precision_test.rb

```diff
@@ -0,0 +1,102 @@
+require 'cases/helper'
+require 'support/schema_dumping_helper'
+
+if ActiveRecord::Base.connection.supports_datetime_with_precision?
+class DateTimePrecisionTest < ActiveRecord::TestCase
```

Comment:
@egilburg no-indent for certain top-level conditionals is the norm for quite a few of our tests, where we're conditionally doing [… big chunk of code …].